scripts/mbedtls_framework/c_wrapper_generator.py

```diff
@@ -65,42 +70,49 @@ def _write_prologue(self, out: typing_util.Writable, header: bool) -> None:
 
         This includes a description comment and some include directives.
         """
-        out.write("""/* Automatically generated by {}, do not edit! */
+        prologue = blockstr_format(f'''
```

Comment:
Continuing https://github.com/Mbed-TLS/mbedtls-framework/pull/25#discussion_r1672223530 — This deintentation approach will fail as soon as we start generating multiline blocks of code inside functions. So it's not future proof and I would prefer not to use it. This is an objective argument, in addition to the subjective argument about readability.

I'm not going to block it because I want to move on. I do think it is a waste of time that this pull request makes a change that we're going to undo later, but I'm not going to insist on reverting that change now.

Reply:
I have rewritten the implementation with a regex single liner which will satisfy that future requirement.